src/Sequence.cc

```diff
@@ -315,26 +315,20 @@ void Sequence::execute(Context& context, CommChannel* comm)
 
     if (exception_thrown)
     {
-        gul14::string_view msg_in{ exception_message };
+        const auto cause = remove_abort_markers_from_error_message(exception_message);
```

Comment:
It took me by surprise that `remove_long_long_name_message()` took the message by reference and as IN/OUT parameter. We use no further optimizations here, so maybe a `pair` or something would be more obvious?

Maybe I'm in a too critical mood ;-D

_Addendum: I would have called that function `remove_abort_markers()`, of course it is from an error message, from what else could that be?_

Reply:
:+1: